Generalized LT/LTV Reduction on V3

eyalovadya opened the pull request:
Discussion: https://governance.aave.com/t/generalized-lt-ltv-reduction-on-aave/16766/4

sendra commented on src/20240324_Multi_GeneralizedLTLTVReductionOnAave/GeneralizedLTLTVReductionOnAave.md:
```diff
@@ -0,0 +1,39 @@
+## Specification
+
+| Assets            | LT           | LTV       | Total Liquidated |
+| ----------------- | ------------ | --------- | ---------------- |
```
Table should be more clear of which configs of tokens on which networks have been updated. As if not i would not know just by reading the table, which are on ethereum or arbitrum etc...

eyalovadya commented:
Pushed the following fixes in this commit - [Fixes](https://github.com/bgd-labs/aave-proposals-v3/pull/281/commits/341db8759a113effd457f1b7c9565bdf94b76ec8):

Specifications table changes: 
Removed Arbitrum  LUSD (cannot be used as collateral)
Removed Optimism LUSD (cannot be used as collateral)
Changed Arbitrum DAI current LT to 82
Changed Polygon DAI current LT to 81 and removed the 76 LTV (same as proposed)

Specifications table and implementation changes:
Removed Avalanche FRAX (can only be used as collateral is isolation mode and LT/LTV are already lower than the proposed)
Removed Ethereum LUSD (LT/LTV are already lower than the proposed)
